docs/ref/models/fields.txt

```diff
@@ -1723,6 +1724,41 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
     integrity, this will cause an :exc:`~django.db.IntegrityError` unless
     you manually add an SQL ``ON DELETE`` constraint to the database field.
 
+* .. attribute:: DB_CASCADE
+
+    .. versionadded:: 5.0
+
+    ``DB_CASCADE`` is a custom option introduced in Django that allows the
+    database to handle cascading deletes instead of relying on Django's
+    emulation of the SQL constraint ``ON DELETE CASCADE``. With ``DB_CASCADE``,
+    the database itself takes care of deleting related objects when a
+    referenced object is deleted.
```

Comment:
As reference documentation I think these can be short descriptions of what the option does. I don't have the precise words to describe this, so take this as inspiration rather than a suggestion. This also applies to the options below as well. 

```suggestion
    Enables the database handling of cascading deletes by adding the ``ON DELETE CASCADE``
    constraint.
```